projects/1.20/assets/ex-deorum/exdeorum/lang/zh_cn.json

```
@@ -0,0 +1,171 @@
+{
+    "advancements.exdeorum.core.barrel.description": "使用木桶将有机物堆肥以形成泥土",
+    "advancements.exdeorum.core.barrel.title": "那是绿色垃圾",
+    "advancements.exdeorum.core.crook.description": "制作一个钩子，用它从树叶上收获更多的树苗。",
+    "advancements.exdeorum.core.crook.title": "给他钩子",
+    "advancements.exdeorum.core.root.description": "在虚空世界中诞生，眼前只有一座小岛",
+    "advancements.exdeorum.core.root.title": "别往下看...",
+    "advancements.exdeorum.core.silk_worm.description": "获得一个蚕，然后用它感染一棵树，得到线",
+    "advancements.exdeorum.core.silk_worm.title": "这看起来可以吃",
+    "advancements.exdeorum.core.string_mesh.description": "制作一个线网，将它放在筛子上以使用。",
```

```suggestion
    "advancements.exdeorum.core.string_mesh.description": "合成一张筛子用的线网",
```

```suggestion
    "advancements.exdeorum.core.string_mesh.description": "合成一张用在筛子上的线网",
```


`string mesh`库里（或者以前版本的无中生有）有其他译法吗
语序优化详见[维基百科：翻译腔](https://zh.m.wikipedia.org/wiki/Wikipedia:%E7%BF%BB%E8%AF%91%E8%85%94)，请全文通查
句末是否加标点请保持全文一致，请全文通查

---

翻译的是丝网，，，我自己用在整合包里用的是丝网没想到昨天上传的是之前的版本，，，

---

呃，确实有的时候记得，有的时候不记得，，，而且我都是按照平常说话的习惯翻的，，，这么严格的吗，，，

---

我们的翻译准则详见[CONTRIBUTING](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/CONTRIBUTING.md)，请一定要认真阅读
有明显的翻译腔痕迹是会被狠狠拷打的

另外查了一下，库里的`string mesh`在exnihilo里（1.12/1.16/1.18）译为`线筛网`